source/mslite.js

```diff
@@ -67,7 +67,7 @@ mslite.Graph = class {
             const data = tensor.data;
             const type = new mslite.TensorType(tensor.dataType, tensor.dims);
             const initializer = (data && data.length > 0) ? new mslite.Tensor(type, tensor.data) : null;
-            return new mslite.Argument(index.toString(), tensor, initializer);
+            return new mslite.Argument(tensor.name, tensor, initializer);
```

Comment:
This will fail for existing models as `tensor.name` might be null?

Reply:
The tensor name will be index.toString() if it is not defined. I also updated the test models.